yt_dlp/extractor/cspan.py

```diff
@@ -242,3 +246,49 @@ def get_text_attr(d, attr):
                 'title': title,
                 'id': 'c' + video_id if video_type == 'clip' else video_id,
             }
+
+
+class CSpanCongressIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?c-span\.org/congress/'
+    _TESTS = [{
+        'url': 'https://www.c-span.org/congress/?chamber=house&date=2017-12-13&t=1513208380',
+        'info_dict': {
+            'id': 'house_2017-12-13',
+            'title': 'Congressional Chronicle - Members of Congress, Hearings and More',
+            'description': 'md5:54c264b7a8f219937987610243305a84',
+            'thumbnail': r're:https://ximage.c-spanvideo.org/.+',
+            'ext': 'mp4'
+        }
+    }]
+
+    def _real_extract(self, url):
+        query = parse_qs(url)
+        if 'chamber' in query:
+            video_id = query['chamber'][0]
+        else:
+            video_id = 'senate'
+        if 'date' in query:
+            video_date = query['date'][0]
+            video_id = video_id + '_' + video_date
+        else:
+            video_date = ''
+        webpage = self._download_webpage(url, video_id)
+        if video_date == '':
+            jwp_date = re.search(r'jwsetup.clipprogdate = \'(?P<date>\d{4}-\d{2}-\d{2})\';', webpage)
+            if jwp_date:
+                video_id = video_id + '_' + jwp_date.group('date')
```

Comment:
avoid `+` ing strings. Use fstrings/%s
```suggestion
                video_id = f'{video_id}_{jwp_date.group("date")}'
```